src/video_core/amdgpu/resource.h

```diff
@@ -204,6 +208,19 @@ struct Image {
         return dst_sel_x | (dst_sel_y << 3) | (dst_sel_z << 6) | (dst_sel_w << 9);
     }
 
+    bool CanSwizzleWithFormat() const {
+        // BGRA
+        if (DstSelect() == 0b111100101110 && GetDataFmt() == DataFormat::Format8_8_8_8) {
+            return true;
```

Comment:
One caveat is that, while I added some more BGRA formats recently, I learned that they aren't supported on NVIDIA for storage. Note the ones with low percents:
<img width="608" alt="format" src="https://github.com/user-attachments/assets/73b8aa97-13a3-4c20-b460-ca980b810388" />

Reply:
Hmm, should we then remove the existing swizzle with view implementation and rely only on the shader system

Reply:
Might be a good idea yeah.